### PR TITLE
Poddubka

### PR DESCRIPTION
**Pull request by Surftrader**
bookstore-spring-boot

### CODE REVIEW

**Comment by Surftrader** [comment]
added registration user

File: lab9/poddubka/bookstore-spring-boot/src/main/java/com/hillel/bookstorespringboot/dto/UserDTO.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package com.hillel.bookstorespringboot.dto;
+
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (dinamio) [inline lab9/poddubka/bookstore-spring-boot/src/main/java/com/hillel/bookstorespringboot/dto/UserDTO.java:5]: Why it's marked as Repository?

File: lab9/poddubka/bookstore-spring-boot/src/main/java/com/hillel/bookstorespringboot/controller/UserController.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@RequestMapping("security")
+public class UserController {
+
+    @Autowired
+    UserDao userDao;
+
+    @Autowired
```
Comment (dinamio) [inline lab9/poddubka/bookstore-spring-boot/src/main/java/com/hillel/bookstorespringboot/controller/UserController.java:20]: This doesn't need to be here, and also, autowired

File: lab9/poddubka/bookstore-spring-boot/src/main/java/com/hillel/bookstorespringboot/controller/BookController.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+//            if (authorDao.findAuthorsByAuthorName(a.getAuthorName()).isEmpty()) {
+//                authorDao.save(new Author(a.getAuthorName()));
+//            }
+//        }
+
+        User user = userDao.findUserByUserName(principal.getName());
+
+        Book book = new Book(bookName, listAuthors, user);//
+        bookDao.save(book);
+        return "redirect:/book/book";
```
Comment (dinamio) [inline lab9/poddubka/bookstore-spring-boot/src/main/java/com/hillel/bookstorespringboot/controller/BookController.java:81]: why book/book, why not book?

**Review by dinamio — APPROVED** [review]
Approved, but see comments

**Pull request merged**